reading-json/reading_json.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+"""reading_json.ipynb
+
+Automatically generated by Colab.
+
+Original file is located at
+    https://colab.research.google.com/drive/1R1M1FIJovBLR3ZasjrHEc1Bfe4SkCRlc
+
+json_file.txt
+"""
+
+import json
+
+data = '{"id": 1, "name": "Zia Khan", "sports": ["Ice Hockey", "Cricket"]}, {"id": 2, "name": "Dennis T", "sports": ["Football", "Basketball"]}'
```

Comment:
There is a file attached to the hw assignment that you can read instead of having to write this as a string.